modules/gapi/include/opencv2/gapi/infer/ie.hpp

```diff
@@ -67,6 +67,19 @@ namespace detail {
         Kind kind;
         bool is_generic;
         IEConfig config;
+
+        ParamDesc(const std::string& model, const std::string& weights, const std::string& device,
+                  const std::vector<std::string>& inputNames,
+                  const std::vector<std::string>& outputNames,
+                  const std::unordered_map<std::string, ConstInput>& constInputs,
+                  std::size_t numIn, std::size_t numOut, Kind kind_, bool isGen,
+                  const IEConfig& config_)
```

Comment:
Why do we need this constructor at all?

Reply:
I mean, the `{}`-thing should work?

Reply:
Overall - I would not expect any changes at the module API level when there is some test work happening.

Reply:
Let's make a call